### ISSUE TITLE
11 vocabulary words and sentences 7/7

### ISSUE THREAD

**Issue by shazahuang** [studydash/cards#181]
## Words:
1. Aesthetically
The new building has little aesthetic value/appeal.
2. Affliction
Malnutrition is one of the common afflictions of the poor.
3. Affluence
In her animal spirits there was an affluence of life 
4. Aggregation
Snowflakes are loose aggregates of ice crystals.
5. Agrarian
This part of the country is mainly agrarian.
6. Aim
I aim to be a millionaire by the time I'm 35.
7. Akin to
Plagiarism is akin to theft.
8. Albeit
He tried, albeit without success.
9. Alert
Parents should be alert to sudden changes in children's behavior.
10. Alliance
NATO is sometimes called the Atlantic Alliance.
11. Allegedly
She was arrested for allegedly stealing a car.

**Comment by r002** [comment]
Good sentences, @shazahuang! 🙂

The only kind of odd sentence that I would rephrase is your sentence for `affluence`.  Normally, in English, you will not say:

> "affluence of life"

as `affluence` means:

![image](https://user-images.githubusercontent.com/45280066/124878348-ffd3e800-df99-11eb-8cd7-25ce3b13c6ee.png)

Instead, here's a sentence you'll more commonly see (basically, a fancy way of saying, _"that person/thing/entity has a ton of money!"_:
> ### _The `affluence` in Taylor Swift's neighborhood is astounding; there is a shiny, brand-new Mercedes-Benz in every driveway!_ 😯

<br />

Good work though!  Keep it up! 👍🚀🌟😀
